styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesGCPServiceAccountSecretManager.java

```diff
@@ -64,12 +64,15 @@
   private static final String STYX_WORKFLOW_SA_JSON_KEY = "styx-wf-sa.json";
   private static final String STYX_WORKFLOW_SA_P12_KEY = "styx-wf-sa.p12";
 
-  private static final EpochProvider DEFAULT_SECRET_EPOCH_PROVIDER =
-      KubernetesGCPServiceAccountSecretManager::smearedDailyEpoch;
+  static final Duration DEFAULT_SECRET_EPOCH_PERIOD = Duration.ofDays(7);
```

Comment:
I would prefer keeping the visibility and using magic numbers in unit test for sanity check. Minor issue though.